GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.cs

```diff
@@ -62,6 +65,8 @@ private void SetShorteningStrategy(string strategy)
                 throw new Exception("Unhandled shortening strategy: " + strategy);
         }
 
+        private static Font AnchorFont = new Font(new ListViewItem().Font, FontStyle.Bold);
```

Comment:
Can you make it non-static and derive from the actual list font? Is it a performance optimization?